src/Pim/index.tsx

```diff
@@ -395,6 +395,25 @@ class Pim extends React.PureComponent {
     const { collectionsAddressBook, collectionsCalendar, collectionsTaskList, addressBookItems, calendarItems, taskListItems } = itemsSelector(this.props);
     const [contacts, events, tasks] = itemValuesSelector(this.props);
 
+    const birthdays = contacts.filter((c) => c.bday).map((c) => {
+      let { year, month, day } = c.bday._time;
+      year = year || '2020';
+      if (month < 10) {
+        month = '0' + month;
+      }
+      if (day < 10) {
+        day = '0' + day;
+      }
+
+      const event = 
+        'BEGIN:VEVENT\n' +
+        `SUMMARY:${c.fn}'s Birthday\n` +
+        `DTSTART;VALUE=DATE:${year}${month}${day}\n` +
+        'RRULE:FREQ=YEARLY\n' +
+        'END:VEVENT';
+      return new EventType(parseString(event));
+    });
```

Comment:
Creating events and parsing them like this is probably not the best way of doing it.
Please just construct the event here, it's really just a matter of doing:
```
const event = new EventType();
event.uid = uuid.v4();
event.summary = `${c.fn}`s Birthday`;
event.startDate = timeNativeToVobject(date);
const rrule: ICAL.RecurData = {
    freq: 'YEARLY',
    interval: 1,
  };
event.rrule = new ICAL.Recur(rrule);
return event;
```
Where `timeNativeToVobject` is https://github.com/etesync/ios/blob/master/src/sync/helpers.ts#L192